test_time_difference_reader.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import torch
+import numpy as np
+from torch.utils.data import DataLoader
+from costar_dataset.block_stacking_reader_torch import CostarBlockStackingDataset
+
+from tqdm import tqdm
+
+if __name__ == '__main__':
+    """ To test block_stacking_reader for feature_modes time_difference_images and cross_modal_embeddings"""
+    
+    args_parser = argparse.ArgumentParser()
+    args_parser.add_argument('--videos_path', help='Path for training data')
```

Comment:
Can you update this to work with the default directory for the dataset? Can you also add a comment at the top with the command line command you used to test it?

There aren't really any comments in this code explaining what is happening and what the options mean. Like what folder level exactly should videos_path point to? Do you need to preprocess the costar dataset to get the videos, or is it really a dataset path?